src/Gateway.php

```diff
@@ -15,6 +15,15 @@ public function __construct ( $host, $request_options = array())
         $this->description = 'Endpoint SDK for using the gateway';
     }
 
+    public function login($credentials) {
+        if (!is_array($credentials))
+            throw new Exception('Login requires first param to be an Associative Array');
```

Comment:
First but also only

Reply:
Copied from the same error in `Api->login()`; I'll update the exception there too.